.gitignore

```diff
@@ -0,0 +1,60 @@
+# Logs
```

Comment:
we're usually ignoring package-lock.json

Reply:
adding it